vmm/src/lib.rs

```diff
@@ -3695,6 +3695,8 @@ mod tests {
             rootfs_file.path().to_str().unwrap()
         );
 
+        // This check is racy; if logger has already been initialized by another test, it is possible
+        // that it w
```

Comment:
This comment just trails off into the ether.

Reply:
Yes I wanted to say that the check would fail if the logger got initialized by another thread when in fact it will not fail as it would give an error. Problem is the error will not be the same every time but it would still be an error.